### PR TITLE
docs(ngIf): Fixed typo

### PR DESCRIPTION
**Pull request by MetalMichael**
"falsy" -> "false"

### CODE REVIEW

**Comment by MetalMichael** [comment]
Submitted CLA

File: src/ng/directive/ngIf.js [hunk 1]
```diff
@@ -39,7 +39,7 @@
  * @element ANY
  * @scope
  * @priority 600
- * @param {expression} ngIf If the {@link guide/expression expression} is falsy then
+ * @param {expression} ngIf If the {@link guide/expression expression} is false then
```
Comment (caitp) [inline src/ng/directive/ngIf.js]: it's not really a typo -- see https://developer.mozilla.org/en-US/docs/Glossary/Falsy

**Comment by pkozlowski-opensource** [comment]
Yup, @caitp is right! We had similar PRs in the past so maybe we should link the `falsy` word to https://developer.mozilla.org/en-US/docs/Glossary/Falsy so people are less likely to be confused by it?

**Comment by pkozlowski-opensource** [comment]
Bah, we've got 7 references to the `falsy` word in docs, not sure if it makes sense to link them all to the definition...

Going to close this one as the docs is correct. If anyone is up to sending PR to link those words to a definition that PRs welcomed!

**Pull request closed without merging**